invenio_audit_logs/records/models.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Base model classes for Audit Logs in Invenio."""
+
+
+from invenio_db import db
+from invenio_records.models import RecordMetadataBase
+from sqlalchemy.types import String
+
+
+class AuditLogModel(db.Model, RecordMetadataBase):
+    """Model class for Audit Log."""
+
+    __tablename__ = "audit_logs_metadata"
+
+    encoder = None
+
+    action = db.Column(String(255), nullable=False)  # TODO: Might change to Enum
```

Comment:
it might change when we have the different audit logs registration?

Reply:
There's also something about space efficiency, so having something like a `SMALLINT` backed by an extensible `Enum` might be a good option... We haven't done anything similar in terms of types registration that directly map to DB columns, so I'm not sure if there's a clear benefit actually or if it will be trivial to implement.